services/server/src/server/controllers/repository/repository.routes.ts

```diff
@@ -60,6 +51,14 @@ router
   .route("/check-all-by-addresses")
   .get(safeHandler(checkAllByChainAndAddressEndpoint));
 
+router
```

Comment:
This is the new alternative to the removed repository static files route https://github.com/ethereum/sourcify/pull/1390/files#diff-c71290f3b0c3ad7664c772a606bb8ec512d83d3f0d41bb73e792cfcbcac569e6L269-L274